pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[build-system]
+requires = [
+    "setuptools >= 51",
+    "wheel",
+    "setuptools_scm[toml] >= 6.2",
+    "setuptools_scm_git_archive",
```

Comment:
Is `setuptools_scm_git_archive` needed?

Reply:
Not strictly, but it's useful if you want to be able to install from the Github tarballs.

Reply:
Although I agree that it's a nice to have feature for devs that checkout fortls and build it themselves, I don't agree it should be listed as a general requirement, since it would also be installed whenever one downloads fortls through PyPi.
Please correct me if I am wrong on this one.